.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## JIRA Ticket
```

Comment:
Could you add a title at the top like 

\# feat(scope): pr title in semver format

Reply:
Actioned